Dockerfile

```diff
@@ -1,18 +1,31 @@
 # read the doc: https://huggingface.co/docs/hub/spaces-sdks-docker
 # you will also find guides on how best to write your Dockerfile
-
-FROM node:19
-
-RUN npm install -g pm2
+FROM node:19 as builder-production
```

Comment:
Any reason not to use node:19-slim for this builder stage ?

Reply:
slim container dont always have everything for build. I dont know here if it will work, but some deps needs gcc etc.

Reply:
Ok fair enough, thanks!